cmd/immuadmin/command/database.go

```diff
@@ -102,9 +112,69 @@ func (cl *commandline) database(cmd *cobra.Command) {
 		},
 		Args: cobra.ExactArgs(1),
 	}
-	addDbUpdateFlags(cc)
+	addDbUpdateFlags(createCmd)
+
+	loadCmd := &cobra.Command{
+		Use:               "load",
+		Short:             "Load database",
+		Example:           "load {database_name}",
+		PersistentPreRunE: cl.ConfigChain(cl.connect),
+		PersistentPostRun: cl.disconnect,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			_, err := cl.immuClient.LoadDatabase(cl.context, &schema.LoadDatabaseRequest{
+				Database: args[0],
+			})
+			if err != nil {
+				return err
+			}
+
+			fmt.Fprintf(cmd.OutOrStdout(), "database '%s' successfully loaded\n", args[0])
+			return nil
+		},
+		Args: cobra.ExactArgs(1),
+	}
+
+	unloadCmd := &cobra.Command{
+		Use:               "unload",
+		Short:             "Unload database",
+		Example:           "unload {database_name}",
+		PersistentPreRunE: cl.ConfigChain(cl.connect),
+		PersistentPostRun: cl.disconnect,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			_, err := cl.immuClient.UnloadDatabase(cl.context, &schema.UnloadDatabaseRequest{
+				Database: args[0],
+			})
+			if err != nil {
+				return err
+			}
+
+			fmt.Fprintf(cmd.OutOrStdout(), "database '%s' successfully unloaded\n", args[0])
+			return nil
+		},
+		Args: cobra.ExactArgs(1),
+	}
 
-	cu := &cobra.Command{
+	deleteCmd := &cobra.Command{
+		Use:               "delete",
+		Short:             "Delete database",
+		Example:           "delete {database_name}",
+		PersistentPreRunE: cl.ConfigChain(cl.connect),
+		PersistentPostRun: cl.disconnect,
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
This is a destructive operation. Maybe we should do some simple safety action here?

Like require an extra argument '--yes-i-know-what-i-am-doing`?